docs/products/opensearch/howto/sample-dataset.rst

```diff
@@ -0,0 +1,169 @@
+Sample dataset
+--------------
+Here you will find different sample datasets for OpenSearch and how to interact with them.
+
+Before exploring the available datasets, we need an OpenSearch server to load the data on. A quick and easy way to get a server is through `Aiven for OpenSearch <https://aiven.io>`_. If don't have an Aiven account yet, `sign up <https://console.aiven.io/signup?utm_source=github&amp;utm_medium=organic&amp;utm_campaign=devportal&amp;utm_content=repo>`_ and enjoy our free trial!
+
+After logging in the Aiven Console, chose the OpenSearch version, select the cloud and plan of your choice, give the service a name and hit the "Create Service" button. In a couple of minutes, you will have an OpenSearch server up and running.
+
+Epicurious recipes
+==================
+A dataset from `Kaggle <https://www.kaggle.com/hugodarwood/epirecipes>`_ with recipes, rating and nutrition information from `Epicurious <https://www.epicurious.com>`_.
+
+Let's take a look at the a sample document:
+
+.. code:: json
+
+    {
+        "title": "A very nice Vegan dish",
+        "desc": "A beautiful description of the recipe",
+        "date": "2015-05-01T04:00:00.000Z",
+        "categories": [
+            "Vegan",
+            "Tree Nut Free",
+            "Soy Free",
+            "No Sugar Added"
+        ],
+        "ingredients": [
+            "list",
+            "of",
+            "ingredients"
+        ],
+        "directions": [
+            "list",
+            "of",
+            "steps",
+            "to prepare the dish"
+        ],
+        "calories": 32.0,
+        "fat": 1.0,
+        "protein": 1.0,
+        "rating": 5.0,
+        "sodium": 959.0,
+    }
+
+
+Load the data
+'''''''''''''
+Follow the steps below to load the sample data into your OpenSearch service:
```

Comment:
I think this needs a heads up that there are other ways to do this but the example uses python to read the data and push it up to OpenSearch.